tests/test_search.py

```diff
@@ -0,0 +1,1036 @@
+import pytest
+import redis
+import bz2
+import csv
+import time
+import os
+import sys
+
+from io import TextIOWrapper
+from .conftest import skip_ifmodversion_lt
+from redis import Redis
+
+import redisplus.search
+from redisplus.client import RedisPlus
```

Comment:
@chayim Is there a way to import all these at once?

Reply:
Nah, this is a nice clean way to do it.